tests/consensus_tests/test_resharding_deferred.py

```
@@ -293,6 +293,22 @@ def test_resharding_transfer_deferred_points(tmp_path: pathlib.Path, direction:
         )
         assert_http_ok(resp)
         time.sleep(1)
+    else:
+        # Without this, the replicas stays in "ReshardingScaleDown" state
+        for shard_id in range(target_shard_id):
+            peer_id, _ = find_replica(shard_id, info, peer_api_uris, peer_ids)
+
+            resp = requests.post(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)